docs/source/elastic/quickstart.rst

```diff
@@ -22,6 +23,7 @@ and at most ``MAX_SIZE`` nodes.
     torchrun
         --nnodes=MIN_SIZE:MAX_SIZE
         --nproc_per_node=TRAINERS_PER_NODE
+        --max_restarts=NUM_ALLOWED_MEMBERSHIP_CHANGE
```

Comment:
should it be `NUM_ALLOWED_MEMBERSHIP_CHANGE_OR_FAILURES`? or is that too verbose

Reply:
yea, you're right updated it to `NUM_ALLOWED_FAILRES_OR_MEMBERSHIP_CHANGES` in V2